src/OpenLoco/src/Map/TileManager.cpp

```diff
@@ -1043,6 +1063,210 @@ namespace OpenLoco::World::TileManager
         });
     }
 
+    // 0x004690FC
+    void setTerrainStyleAsCleared(const Pos2& pos)
+    {
+        auto* surface = World::TileManager::get(pos).surface();
+        if (surface == nullptr)
+        {
+            return;
+        }
+        if (surface->isIndustrial())
+        {
+            return;
+        }
+        if (surface->var_6_SLR5() > 0)
+        {
+            surface->setVar6SLR5(0);
+            surface->setVar4SLR5(0);
+
+            Ui::ViewportManager::invalidate(pos, surface->baseHeight(), surface->baseHeight() + 32, ZoomLevel::eighth);
+        }
+        if (surface->var_4_E0() > 0)
+        {
+            surface->setVar4SLR5(0);
+
+            Ui::ViewportManager::invalidate(pos, surface->baseHeight(), surface->baseHeight() + 32, ZoomLevel::eighth);
+        }
+    }
+
+    // 0x00468651
+    uint32_t adjustSurfaceHeight(World::Pos2 pos, SmallZ targetBaseZ, uint8_t slopeFlags, std::set<World::Pos3, LessThanPos3>& removedBuildings, uint8_t flags)
```

Comment:
Bit of a misnomer, given it doesn't actually adjust the height. Maybe `canAdjustSurfaceHeight` would be better? Any thoughts?